server/middleware.js

```diff
@@ -88,122 +87,29 @@ module.exports.flux = {
       // Fetch from localhost.
       fetchConversions(types, value)
         .then(function (conversions) {
+
           // Bootstrap, snapshot data to res.locals and flush for next request.
-          flux.bootstrap(JSON.stringify({
-            ConvertStore: {
-              conversions: conversions,
-              types: types,
-              value: value
-            }
-          }));
+          var store = createStore({
+            conversions: conversions,
+            types: types,
+            value: value
+          });
 
           // Stash bootstrap, and _fully-rendered-page_ with proper data.
-          res.locals.bootstrapData = flux.takeSnapshot();
+          res.locals.bootstrapData = store.getState();
           if (req.query.__mode !== "noss") {
```

Comment:
This was a bug / oversight in the original code -- At https://github.com/FormidableLabs/converter-react/blob/chore-redux/server/middleware.js#L79 we already check:

```js
if (req.query.__mode === "noss") { return next(); }
```

Thus this conditional can be removed because `req.query.__mode !== "noss"` is **always true**.